config/app.php

```diff
@@ -133,7 +133,7 @@
 
 	'key' => env('APP_KEY'),
 
-	'cipher' => 'AES-256-CBC',
+	'cipher' => env('APP_CIPHER', 'AES-256-CBC'),
```

Comment:
I'm not sure how much benefit there is to exposing this, but I guess there's no harm.